src/ir_Hitachi.h

```diff
@@ -347,6 +370,9 @@ class IRHitachiAc3 {
   void stateReset(void);
 #if SEND_HITACHI_AC3
   void send(const uint16_t repeat = kHitachiAcDefaultRepeat);
+  /// Run the calibration to calculate uSec timing offsets for this platform.
+  /// @note This will produce a 65ms IR signal pulse at 38kHz.
+  ///   Only ever needs to be run once per object instantiation, if at all.
   int8_t calibrate(void) { return _irsend.calibrate(); }
```

Comment:
Add this to all files at once as well

Reply:
Ack. I'll think about it too. Again, maybe something for a base class.